README.md

```diff
@@ -122,14 +119,44 @@ This step will prompt you to enter 2 details -
 
 ### About Env Configs
 
-For ease of development and deployment there are configs in /Hyperswitch-react-demo-app/.env
+For ease of development and deployment there are configs in /Hyperswitch-react-demo-app/.env -
+
+#### How to Find Hyperswitch Publishable Key and API Key
+
+To integrate Hyperswitch into your project, follow these steps:
+
+1. **Register**: Begin by registering at [Hyperswitch](https://app.hyperswitch.io/register) to gain access to your dashboard.
+
+2. **Access Your Dashboard**: Once logged in, navigate to your dashboard's home page.
+
+3. **Find Your Keys**: In the sidebar, locate the "Developers" section, then click on [API Keys](https://app.hyperswitch.io/developer-api-keys). Here, you can generate your API Key. Additionally, your Publishable Key is available within the same section.
+
+4. **Integration**: After obtaining your keys, add them to your project's environment variables or configuration file for seamless integration.
+
+The above method is for using our Sandbox Environment.
+
+If you are running our backend locally, you can use our [Postman Collection](https://github.com/juspay/hyperswitch/blob/main/docs/try_local_system.md#try-out-our-apis) for generating the API Key and Publishable Key.
 
 - `HYPERSWITCH_PUBLISHABLE_KEY` - Publishable key of your Hyperswitch Account
-- `HYPERSWITCH_SECRET_KEY` - Api key of your Hyperswitch Account
+- `HYPERSWITCH_SECRET_KEY` - API key of your Hyperswitch Account
 - `HYPERSWITCH_SERVER_URL` - URL of your hosted Hyperswitch Backend server
 - `HYPERSWITCH_CLIENT_URL` - URL of your hosted Hyperswitch SDK
 - `SELF_SERVER_URL` - URL of your node server (/Hyperswitch-react-demo-app/server.js)
```

Comment:
Can we at least include values for these when running the web client and React demo app locally, and trying to access either the sandbox or locally running app server?

Would make it easier for people to copy-paste when running the web client and demo app locally.

Another option is to have the prompt script display default values for these and use the defaults if nothing is specified by the user.

Reply:
We shouldn't give default value of these because if someone try to do a payment where they will see the payment history. I guess the steps are clear now a bit wh
CC: @akash-kamble-juspay

Reply:
I think we should have sbx values as default. I see no harm except we getting some additional logs. But at least this reduces the friction for someone to try out our product.

Reply:
Sure. Can we create one dummy account for the same and add the keys for now? Then we need to change the script that enters the values to env and stop that I guess

Reply:
We have given the dedicated steps to create the API key and Publishable key so Dummy keys won't make much sense I guess just to start the SDK won't be good they might be able to see the Payment details if want to do so too.

Reply:
> We shouldn't give default value of these because if someone try to do a payment where they will see the payment history.

@PritishBudhiraja @akash-kamble-juspay I'm not at all suggesting that we should provide/include values for Hyperswitch sandbox API keys or publishable keys. I'm only suggesting that we document the URLs that one can use when running the demo app against either the sandbox environment, or when running a server locally.

The review comment is scoped to only three lines corresponding to the URLs. Of course, providing sandbox API keys is a bad idea.